cherrypy/_cpdispatch.py

```diff
@@ -203,15 +207,18 @@ def test_callable_spec(callable, callable_args, callable_kwargs):
     import inspect
 except ImportError:
     def test_callable_spec(callable, args, kwargs):  # noqa: F811
+        """Test callable spec."""
         return None
 else:
     def getargspec(callable):
+        """Get arg spec."""
         return inspect.getfullargspec(callable)[:4]
 
 
 class LateParamPageHandler(PageHandler):
+    """LateParamPageHandler PageHandler class.
```

Comment:
Let's invent something more useful than repeating all identifiers from the previous line...

Reply:
```suggestion
    """Page handler callable with delayed request parameters binding.
```